static/admin/config.yml

```diff
@@ -21,20 +21,59 @@ collections:
           - { label: Facebook Link, name: facebook, widget: string }
           - { label: LinkedIn Link, name: linkedIn, widget: string }
           - { label: Github Link, name: github, widget: string }
+          - { label: TikTok Link, name: tiktok, widget: string }
 
       - label: "Index"
         name: "index"
         file: "/src/markdown/index.md"
         fields:
           - { label: Heading, name: heading, widget: string }
           - { label: Image, name: image, widget: image }
-          - {
-              label: SEO Page Description,
-              name: meta_page_description,
-              widget: text,
-              required: false,
-              hint: "This text will show up in the Google description of the page (The text under the link)",
-            }
+          - label: SEO Page Description
+            name: meta_page_description
+            required: false
+            hint: "This text will show up in the Google description of the page (The text under the link)"
+          - label: Recruitment Open
+            name: recruitment_open
+            widget: boolean
+            hint: Toggle this on when recruitments are open.
+          - label: Recruitment Link
+            name: recruitment_link
+            widget: string
+          - label: Recruitment Info Link
+            name: recruitment_info
+            widget: string
+          - label: Recruitment Description
+            name: recruitment_description
+            widget: text
+          - label: Splash Section
+            name: splash
+            widget: object
+            hint: Splash section, inclusing video
+            fields:
+              - { label: Splash Heading, name: heading, widget: string }
+              - { label: Splash Body, name: body, widget: string }
+              - label: Trailer Embed Link
+                name: trailer_link
+                widget: string
+                hint: This link MUST be an embed link e.g. https://youtube.com/embed/someVideoId, otherwise it may not work.
+          - label: Sponsor Section
+            name: sponsors
+            widget: list
+            fields:
+              - { label: Name, name: name, widget: string }
+              - { label: Link to Sponsor, name: link, widget: string }
+              - { label: Image, name: image, widget: image }
+
+      - label: "About Page"
+        name: "about"
+        file: "/src/markdown/about.md"
+        fields:
+          - { label: Heading, name: heading, widget: string }
+          - label: SEO Page Description
+            name: meta_page_description
+            required: false
+            hint: "This text will show up in the Google description of the page (The text under the link)"
```

Comment:
Do we need to repeat this for every page? Wouldn't it be the same on google?

Reply:
I think the page description is specific to each page though I haven't tested it yet, so it allows to set a different page description for each page. If we want to set a global page description I might be able to do that on another PR.